dotcom-rendering/src/layouts/FrontLayout.tsx

```diff
@@ -103,16 +111,17 @@ const decideAdSlot = (
 
 export const FrontLayout = ({ front, NAV }: Props) => {
 	const {
-		config: { isPaidContent },
+		config: { isPaidContent, abTests },
 	} = front;
 
-	const isInEuropeTest =
-		front.config.abTests.europeNetworkFrontVariant === 'variant';
+	const isInEuropeTest = abTests.europeNetworkFrontVariant === 'variant';
+
+	const theme = isPaidContent ? ArticleSpecial.Labs : ArticlePillar.News;
```

Comment:
Setting the `theme` is required to ensure the nav bar [correctly displays the roundel](https://github.com/guardian/dotcom-rendering/blob/e65e2d33014a7b1a330fa8ba50c396a51dd314fe/dotcom-rendering/src/components/Nav/Nav.tsx#L62-L64) on paid fronts.

Reply:
If the variable isn’t use anywhere else, you may want to place this ternary inside the format object directly.